containerized.mk

```diff
@@ -4,6 +4,26 @@ DOCKER_IMAGE_DIR=${GOPATH}/src/${PROJECT_ROOT}
 
 DOCKER_SWARMKIT_DELVE_PORT ?= 2345
 
+# HOW TO USE CONTAINERIZED BUILDS:
+#
+# There are lots of flags that we've added to containerized builds of swarmkit,
+# mostly to support the various swarmkit dev's workflows. These can be kind of
+# confusing, and might feel like they require reading the makefile to fully
+# understand. For clarity, these are the various flags and what they do:
+#
+# DOCKER_SWARMKIT_USE_CONTAINER: Build swarmkit inside of a docker container.
+# DOCKER_SWARMKIT_NOMOUNT: Runs the make target without bindmounting the local
+# 	directory into the container. Useful if you're, say, running tests over a
+# 	network socket. Otherwise, we'll be doing the equivalent of 
```

Comment:
Might be worth clarifying that you mean talking to the engine running the dev container over a TCP socket?

Reply:
well, strictly speaking, it doesn't have to be a TCP socket. in the past, i've forwarded a unix socket over SSH.